internal/websocket/ws.go

```diff
@@ -359,37 +372,38 @@ func (ws *Websocket) dial(ctx context.Context, mode ipMode) (*nhws.Conn, *http.R
 }
 
 type custRT struct {
-	transport http.Transport
+	transport *http.Transport
```

Comment:
tooltip started complaining about the non-pointer http.Transport: `literal copies lock value from *transport: net/http.Transport contains sync.Mutexcopylocksdefault`